frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationCard.tsx

```diff
@@ -496,12 +498,23 @@ export const AlertConfigurationCard = ({
 		}),
 	)
 
-	const emails = emailSuggestions.map((email) => ({
+	const emails = uniq([
+		...emailSuggestions,
+		...form.getFieldValue('emails'),
```

Comment:
looks like this is crashing on existing alerts
https://www.loom.com/share/66ce26ec59d74a4aa543d9b227369074
https://preview.highlight.io/1/errors/SL0xPKILXVgIQcCI9nc7Nc0RWbjO?page=1

Reply:
Thanks for catching this, @Vadman97!

Reply:
no problem! still seems to be crashing after 0e22b40, though I'm guessing you know since you marked it as a draft.

Reply:
@Vadman97 something odd here is that I'm not seeing this error on the Render preview, only on the Reflame preview. I also see the crash when I visit on a Reflame preview for another PR. E.g. https://github.com/highlight/highlight/pull/5786/checks?check_run_id=14665975113

@lewisl9029 - any ideas why we might be seeing these crashes only in Reflame previews?